parsers/s01-parse/corvese/apache-guacamole-logs.yaml

```diff
@@ -0,0 +1,18 @@
+name: corvese/apache-guacamole-logs
+description: "Parses Apache Guacamole logs"
+filter: "evt.Parsed.program == 'apache-guacamole'"
+onsuccess: next_stage
+nodes:
+  - grok:
+      pattern: '%{HOUR:hour}:%{MINUTE:minute}:%{SECOND:second}.*Authentication attempt from \[%{IP:source_ip}.*for user "%{USERNAME:username}" failed'
```

Comment:
Change `%{HOUR:hour}:%{MINUTE:minute}:%{SECOND:second}` this to %{TIME:timestamp}

Reply:
Addressed in 8b55fe57370df13baa5310b8235720f9e07b143e